### PR TITLE
Fix for Undeploy Workflow Empty Matrix

### LINKED ISSUE #134: Environment Cleanup Failing due to Misconfigured `undeploy-*` Matrix
See https://github.com/ACCESS-NRI/ACCESS-OM3/actions/runs/10988249190/job/30504356713#step:3:2

This is due to the `jq` used to create the matrix https://github.com/ACCESS-NRI/build-cd/blob/main/.github/workflows/undeploy-1-setup.yml#L26 not turning the overall result into array. 

Do the above and this should allow the matrix job to proceed.

### PR DESCRIPTION
**Pull request by CodeGat**
In this PR:
- Enclose matrix input in array so it is correctly interpreted later in the pipeline

Closes #134

### CODE REVIEW

File: config/settings.json [hunk 1]
```diff
@@ -13,6 +13,9 @@
           "0.22": "21da7d7e2b5e2680cd9d2e0a2fb4a7d13d8baa9d"
         }
       }
+    },
+    "Setonix": {
+
```
Comment (aidanheerdegen) [inline config/settings.json:18]: 👀 

It's ok that this is empty?

**Review by aidanheerdegen — DISMISSED** [review]
&langle; waves hands airily &rangle;

Looks good to me

File: config/settings.json (hunk 1 above)
Comment (CodeGat) [reply config/settings.json:18]: WHOOPS that was for testing.

Comment (CodeGat) [reply config/settings.json]: Removed as part of https://github.com/ACCESS-NRI/build-cd/pull/137/commits/7daa1eccdfb69e26528129e5a41f6d414a13fe07

**Review by aidanheerdegen — APPROVED** [review]
Looks good-er to me.

**Pull request merged**